##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD build --keep-empty-dirs --no-resave-data ReUseData
###
##############################################################################
##############################################################################


* checking for file ‘ReUseData/DESCRIPTION’ ... OK
* preparing ‘ReUseData’:
* checking DESCRIPTION meta-information ... OK
* installing the package to build vignettes
* creating vignettes ... ERROR
--- re-building ‘ReUseData_data.Rmd’ using rmarkdown
Quitting from lines 92-98 (ReUseData_data.Rmd) 
Error: processing vignette 'ReUseData_data.Rmd' failed with diagnostics:
NA/NaN argument
--- failed re-building ‘ReUseData_data.Rmd’

--- re-building ‘ReUseData_quickStart.Rmd’ using rmarkdown
Traceback (most recent call last):
  File "/home/biocbuild/.cache/R/basilisk/1.11.2/Rcwl/1.15.5/env_Rcwl/bin/cwltool", line 11, in <module>
    sys.exit(run())
  File "cwltool/main.py", line 1518, in run
  File "cwltool/main.py", line 1061, in main
  File "cwltool/utils.py", line 127, in versionstring
  File "/home/biocbuild/.cache/R/basilisk/1.11.2/Rcwl/1.15.5/env_Rcwl/lib/python3.8/site-packages/pkg_resources/__init__.py", line 956, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/home/biocbuild/.cache/R/basilisk/1.11.2/Rcwl/1.15.5/env_Rcwl/lib/python3.8/site-packages/pkg_resources/__init__.py", line 815, in resolve
    dist = self._resolve_dist(
  File "/home/biocbuild/.cache/R/basilisk/1.11.2/Rcwl/1.15.5/env_Rcwl/lib/python3.8/site-packages/pkg_resources/__init__.py", line 861, in _resolve_dist
    raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.ContextualVersionConflict: (cwl-utils 0.19 (/home/biocbuild/.cache/R/basilisk/1.11.2/Rcwl/1.15.5/env_Rcwl/lib/python3.8/site-packages), Requirement.parse('cwl-utils>=0.22'), {'cwltool'})
Quitting from lines 76-85 (ReUseData_quickStart.Rmd) 
Error: processing vignette 'ReUseData_quickStart.Rmd' failed with diagnostics:
argument is of length zero
--- failed re-building ‘ReUseData_quickStart.Rmd’

--- re-building ‘ReUseData_recipe.Rmd’ using rmarkdown
Traceback (most recent call last):
  File "/home/biocbuild/.cache/R/basilisk/1.11.2/Rcwl/1.15.5/env_Rcwl/bin/cwltool", line 11, in <module>
    sys.exit(run())
  File "cwltool/main.py", line 1518, in run
  File "cwltool/main.py", line 1061, in main
  File "cwltool/utils.py", line 127, in versionstring
  File "/home/biocbuild/.cache/R/basilisk/1.11.2/Rcwl/1.15.5/env_Rcwl/lib/python3.8/site-packages/pkg_resources/__init__.py", line 956, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/home/biocbuild/.cache/R/basilisk/1.11.2/Rcwl/1.15.5/env_Rcwl/lib/python3.8/site-packages/pkg_resources/__init__.py", line 815, in resolve
    dist = self._resolve_dist(
  File "/home/biocbuild/.cache/R/basilisk/1.11.2/Rcwl/1.15.5/env_Rcwl/lib/python3.8/site-packages/pkg_resources/__init__.py", line 861, in _resolve_dist
    raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.ContextualVersionConflict: (cwl-utils 0.19 (/home/biocbuild/.cache/R/basilisk/1.11.2/Rcwl/1.15.5/env_Rcwl/lib/python3.8/site-packages), Requirement.parse('cwl-utils>=0.22'), {'cwltool'})
Quitting from lines 166-172 (ReUseData_recipe.Rmd) 
Error: processing vignette 'ReUseData_recipe.Rmd' failed with diagnostics:
argument is of length zero
--- failed re-building ‘ReUseData_recipe.Rmd’

SUMMARY: processing the following files failed:
  ‘ReUseData_data.Rmd’ ‘ReUseData_quickStart.Rmd’
  ‘ReUseData_recipe.Rmd’

Error: Vignette re-building failed.
Execution halted
